web_poet/__init__.py

```diff
@@ -1,5 +1,5 @@
 from .fields import field, item_from_fields, item_from_fields_sync
-from .overrides import OverrideRule, PageObjectRegistry, consume_modules
+from .overrides import ApplyRule, OverrideRule, PageObjectRegistry, consume_modules
```

Comment:
What do you think about renaming the module? e.g. rules.py?

Reply:
I was thinking the same thing! 👍

Reply:
Done in https://github.com/scrapinghub/web-poet/pull/84/commits/01112e7e316ea2756a8692b752d757aeae280f23.